src/adaptors/tcp/tcp_adaptor.c

```diff
@@ -2420,6 +2431,17 @@ QD_EXPORT void qd_dispatch_delete_tcp_connector(qd_dispatch_t *qd, void *impl)
         connector->core_conn = 0;
         qd_connection_counter_dec(QD_PROTOCOL_TCP);
         //
+        // Initiate termination of existing connections
+        //
+        if (connector->adaptor_config->terminate_conns) {
+            qd_tcp_connection_t *conn;
+            conn = DEQ_HEAD(listener->connections);
```

Comment:
This should be `conn = DEQ_HEAD(connector->connections)'  not `conn = DEQ_HEAD(listener->connections)`?

Reply:
Ops, copy-paste issue.